src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -83,13 +83,13 @@ const SearchResults = () => {
     const filter = searchParams.get("filter"),
         query = searchParams.get("query"),
         sort = searchParams.get("sort");
-
+    const profile = searchParams.get("profile")?.toUpperCase()
```

Comment:
small syntax improvement - continue off of the const variable definitions started on line 83. you can delete the semi-colon on line 85 and add a comma to define another const variable